code/modules/power/cell.dm

```diff
@@ -103,10 +103,10 @@
 	return (FIRELOSS)
 
 /obj/item/stock_parts/cell/on_reagent_change(changetype)
-	rigged = !isnull(reagents.has_reagent(/datum/reagent/toxin/plasma, 5)) //has_reagent returns the reagent datum
+	if (reagents.has_reagent(/datum/reagent/toxin/plasma, 5))
+		rigged = TRUE //has_reagent returns 0 (FALSE) if there is no reagent

```

Comment:
```suggestion
	rigged = reagents.has_reagent(/datum/reagent/toxin/plasma, 5))

```
might wanna make sure that you can't use this to un-rig cells that have been rigged by means other than a plasma injection, though